File: Estrutura_Condicional/C++/Ex1037/main.cpp
#include <iostream>

int main()
{

    float v;
    std::cin >> v;

    if (v >= 0 and v <= 25){
        std::cout << "Intervalo [0,25]\n";
    } else if (v > 25 and v <= 50){
        std::cout << "Intervalo (25,50]\n";
    } else if (v > 25 and v <= 50) {
        std::cout << "Intervalo (50,75]\n";
    } else if (v > 75 and v <= 100){
        std::cout << "Intervalo (75,100]\n";
    } else {
        std::cout << "Fora de intervalo\n";
    }
        return 0;
}

File: Estrutura_Condicional/C#/Ex1037/Main.cs
using System;

namespace MyCompiler {
    class Program {
        public static void Main(string[] args) {

            float v = float.Parse(Console.ReadLine());
            
            if (v >= 0 && v <= 25){
            Console.WriteLine("Intervalo [0,25]");
            } else if (v > 25 && v <= 50){
            Console.WriteLine("Intervalo (25,50]");
            } else if (v > 75 && v <= 100){
            Console.WriteLine("Intervalo (75,100]");
            } else {
            Console.WriteLine("Fora de intervalo");
            }
            
        }
    }
}